store/postgres/src/relational_queries.rs

```diff
@@ -2818,14 +2832,26 @@ impl<'a> QueryFragment<Pg> for CopyEntityDataQuery<'a> {
         }
         out.push_sql("block_range from ");
         out.push_sql(self.src.qualified_name.as_str());
+        out.push_sql(" where vid >= ");
+        out.push_bind_param::<BigInt, _>(&self.first_vid)?;
+        out.push_sql(" and vid < ");
+        out.push_bind_param::<BigInt, _>(&self.last_vid)?;
```

Comment:
`last_vid` isn't really the last vid but the one more than the last, I think this would be more readable if this where `<= last_vid` and `last_vid` was passed in as the actual last vid.

Reply:
True; I changed that.